agenthub/monologue_agent/utils/prompts.py

```diff
@@ -151,12 +155,30 @@ def parse_action_response(response: str) -> Action:
     Returns:
     - Action: The action that was found in the response string
     """
-    action_dict = json.loads(response)
-    if 'content' in action_dict:
+    json_start = response.find("{")
+    json_end = response.rfind("}") + 1
+    response_json_str = response[json_start:json_end]
```

Comment:
Merge issue--I believe this was moved into json.py

Reply:
good catch, thanks!